### PR TITLE
chore: PACT-1241 on prem release version 1.26.0

### PR DESCRIPTION
**Pull request by Inksprout**

### CODE REVIEW

File: website/docs/docs/on-premises/releases/1.26.0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+## Release date
+
+07 August 2023
+
+## Features
+
+* Support application/problem+json error response format for all non-200 responses
+* null
```
Comment (Ilia) [inline website/docs/docs/on-premises/releases/1.26.0.md:12]: bah! i need to fix those in the script, but I think for the moment, we just clean this up

Comment (bethesque) [reply website/docs/docs/on-premises/releases/1.26.0.md:12]: I'm all for releasing null.

**Comment by bethesque** [comment]
Can you add this into the release notes please Candy https://github.com/pactflow/docs.pactflow.io/pull/253

**Comment by Ilia** [comment]
We may need to rerun the script (unless there is a bug) due to it not creating a release in Jira, @Inksprout did you run it with `-r 1`? my fault for not keeping up with the docs :(

**Pull request closed without merging**